packages/core/types/index.js

```diff
@@ -220,6 +220,7 @@ export interface PluginOptions {
   +autoinstall: boolean;
   +logLevel: LogLevel;
   +rootDir: FilePath;
+  +distDir: FilePath;
```

Comment:
I would say this property should rather be on the `ServerOptions` to not clearly separate "server root" from the "bundle output dist dir per target".

Reply:
I know that the I need a distDir to server from here: https://github.com/parcel-bundler/parcel/blob/edab2589cfe689d009d428165e6188afe563699f/packages/reporters/dev-server/src/ServerReporter.js#L30

@devongovett  what do you think on this?

Reply:
@devongovett ?

Reply:
🤷 I think it's ok... it does reflect the original distDir option that's passed to the API.